diff --git a/src/openMVG/matching/metric.hpp b/src/openMVG/matching/metric.hpp
index 17257cf..54c6d31 100644
--- a/src/openMVG/matching/metric.hpp
+++ b/src/openMVG/matching/metric.hpp
@@ -9,7 +9,7 @@
 #ifndef OPENMVG_MATCHING_METRIC_HPP
 #define OPENMVG_MATCHING_METRIC_HPP
 
-#include "openMVG/matching/metric_simd.hpp"
+// #include "openMVG/matching/metric_simd.hpp"
 #include "openMVG/matching/metric_hamming.hpp"
 #include "openMVG/numeric/accumulator_trait.hpp"
 #include <cstdint>
